CSC8503/TutorialGame.cpp

```diff
@@ -549,33 +532,33 @@ GameObject* TutorialGame::AddBonusToWorld(const Vector3& position, const std::st
 	return apple;
 }
 
-GameObject* TutorialGame::AddAnimationTest(const Vector3& position, const std::string& objectName)
-{
-	float meshSize = 1.0f;
-	float inverseMass = 0.5f;
-
-	GameObject* animTest = new GameObject();
```

Comment:
We won't be using tutorial game. Will your code still work?

Reply:
Nope, I will delete it in next time.